generators/app/templates/config/index.ejs

```diff
@@ -41,6 +41,11 @@ const config = {
     session: {
       header_name: 'authorization',
       secret: process.env.NODE_API_SESSION_SECRET
+    },
+    headers: {
+      apiDate: 'X-API-Date',
+      packageVersion: 'X-Package-Version',
+      nodeVersion: 'X-Node-Version'
```

Comment:
We need this as default value, but i want make configurable through environment variables without change the code, what do you think ?